app/templates/events/trackVolunteers.html

```diff
@@ -248,43 +246,27 @@ <h5 class="modal-title" id="exampleModalLabel">{{modalTitle}}</h5>
             <i class="bi bi-search ms-0"></i>
           </button>
         </div>
-        <br>
-        <h5>Selected Volunteers</h5>
-        <ul class="list-unstyled" id= "addVolunteerList">
+        <ul class="list-unstyled mx-3 mt-2" id= "addVolunteerList">
+          {% if event.isRecurring and not event.isFirstRecurringEvent%}
+            {% if recurringVolunteers | count != 0  %}
+              {%for participant in recurringVolunteers%}     
+                <li>
+                  <input class="recurringVolunteer" type="checkbox" id ="recurringVolunteerCheckbox_{{loop.index}}" name="volunteer[]" value= {{participant}} />
+                  <label class="form-check-label" for="recurringVolunteerCheckbox_{{loop.index}}"> {{participant.firstName}} {{participant.lastName}}</label>
+                  <label class="text-dark" data-toggle="previousVolunteerHover">*</label>
+                </li>
+              {% endfor %}
+            {% endif %}
+          {% endif %}
         </ul>
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-secondary me-auto" data-bs-dismiss="modal">Close</button>
+        <button type="button" id = "selectAllVolunteers" class="btn btn-primary" >Select All</button>
```

Comment:
Everything is looking good. One final thing is, lets only show the `Select All` button when it can be used. This means it should only appear when it is a recurring event and there are previous volunteers to add.